pkg/controller/plan/migration.go

```diff
@@ -416,7 +416,12 @@ func (r *Migration) CleanUp(vm *plan.VMStatus) (err error) {
 		return
 	}
 	if vm.Warm != nil {
-		_ = r.provider.RemoveSnapshots(vm.Ref, vm.Warm.Precopies)
+		if errLocal := r.provider.RemoveSnapshots(vm.Ref, vm.Warm.Precopies); errLocal != nil {
+			r.Log.Info(
```

Comment:
yeah, I wouldn't fail the cleanup on this but not sure about the 'info' level here - I'd expect this to be logged as a warning if we had such a level, so maybe with 'error' level?

Reply:
Error might be little too much, but OK.